pkg/controllermanager/apis/config/types.go

```diff
@@ -95,6 +97,15 @@ type PlantControllerConfiguration struct {
 	SyncPeriod metav1.Duration
 }
 
+// EventControllerConfiguration defines the configuration of the Event controller.
+type EventControllerConfiguration struct {
+	// ConcurrentSyncs is the number of workers used for the controller to work on
+	// events (defaults to 5).
+	ConcurrentSyncs *int
```

Comment:
```suggestion
	ConcurrentSyncs int
```

Imo, we should be consistent with the other configuration structs. It's not a problem to introduce it as a required field either because the `EventControllerConfiguration` struct itself is optional in `ControllerManagerControllerConfiguration`.

/cc @rfranzke

Reply:
I see the advantage of consistency. But for me this is a number that should have a reasonable default. The user can hardly guess which number is appropriate here - so the average user should stick to the default and adapt if running into performance problems. That said, I will adapt it now to your suggestion but would still like to hear your opinion.